library/junos_install_config

```diff
@@ -250,6 +261,9 @@ def _load_via_netconf(module):
         logging.error(msg)
         module.fail_json(msg=msg)
 
+    if args['ignore_warning'] in BOOLEANS + ["True", "False"]:
```

Comment:
Ansible boolean values are normally case-insensitive, but this is not. Suggest changing to:
```
if str(args['ignore_warning']).lower() in BOOLEANS + ["true", "false"]:
```